src/Sentry/Protocol/ISpan.cs

```diff
@@ -8,9 +8,22 @@ namespace Sentry.Protocol
     public interface ISpan : ISpanContext, IHasTags, IHasExtra
     {
         /// <summary>
-        /// Description.
+        /// Span description.
         /// </summary>
-        string? Description { get; set; }
+        // 'new' because it adds a setter.
+        new string? Description { get; set; }
```

Comment:
`new` is yikes

Reply:
yeah, if the reference held is of a base type this `new` won't show up. And the wrong getter will run. But hey, not the worst thing we got going on to add this API right?

Reply:
The getter will run correctly (since both interfaces still map to the same member on the class). However, if you implement explicitly then yeah you can end up with two different properties:

```csharp
ISpanContext.Description {get;set;}
ISpan.Description {get;set;}
```